components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -1,16 +1,235 @@
 <template>
-  <article class="bx--grid page-section">
+  <article class="bx--grid page-section qiskit-capabilities-section">
     <h2>What Can Qiskit Do</h2>
-    <CodeCell :active-blocks="activeBlocks" />
+    <div class="qiskit-capabilities-section__container">
+      <div class="qiskit-capabilities-section__capabilities">
+        <div class="qiskit-capabilities-section__content">
+          <QiskitCapabilityCard
+            v-for="item in capabilities"
+            :id="item.title"
+            :key="item.title"
+            class="qiskit-capabilities-section__card scrollable"
+            :title="item.title"
+            :description="item.description"
+            :thumbnail-resource="item.thumbnailResource"
+            :link="item.link"
+          />
+        </div>
+        <div class="qiskit-capabilities-section__scrolling-ui">
+          <CodeCell :active-blocks="capabilities.map((item, index) => isActiveSection(item, index))" />
+        </div>
+      </div>
+    </div>
+    <div class="qiskit-capabilities-section__ctas">
+      <AppCta
+        class="qiskit-capabilities-section__ctas__link"
+        v-bind="quantumLabCTA"
+        kind="ghost"
+      />
+      <cv-button
```

Comment:
This looks quite similar to the **AppCta** component.

The CSS interaction effects are duplicated here to achieve a similar look and feel, which isn't ideal, but I guess that the **AppCta** isn't currently adaptable enough.

I would consider opening a ticket to either extend the *AppCta* or create a new button component that reuses the CSS styles. It might also be implemented as a web component in our new library. Do you agree with this?

Reply:
You're right about this, Eddy... I discussed this with Goyo for a bit, and decided to submit a PR with this first attempt;

however, we both agree that extending this component is the right way.

I've created a ticket here: https://github.com/Qiskit/qiskit.org/issues/2364